Ogury/OguryAdapterConfiguration.m

```diff
@@ -65,6 +66,11 @@ - (void)initializeNetworkWithConfiguration:(NSDictionary<NSString *, id> * _Null
         return;
     }
 
+    MPConsentStatus mopubConsentStatus = [MoPub sharedInstance].currentConsentStatus;
```

Comment:
Should I perform this check every time an ad is requested or doing only this once is enough?

Reply:
User normally has the right to change his consent any time he wants, and we dont have access to this information so it is safer to keep it like this.

Reply:
That's why I'm asking, this call will be made only once, at the beginning of the adapter.

If, after the initialization, the user changes its consent, the NoTC will not be called unless the adapter is initialized again meaning that any change of consent won't be detected.

Reply:
Since the session persistence will now be in effect, I will move this code to perform the NoTC check & send at every ad request.

This way, we'll always have the latest consent from MoPub.

Reply:
Done.